### PR TITLE
Add all cases in switch statement for graphics pipeline fullscreen quads type

### PR DESCRIPTION
**Pull request by angela28chen**

### CODE REVIEW

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 1]
```diff
@@ -1696,6 +1696,12 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
             frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[1], 3, &colorValues);
             break;
         }
+        case FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_TEXTURE: {
```
Comment (hysw) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:1699]: fyi, the reason for adding bracket (i.e. `case ... : { ... }`) to case statement is to limit variable scope, it's not necessary to add it to a break statement. It's also okay to leave it there if you prefer it

**Review by Keenuts — APPROVED** [review]
Nothing to add outside of the nit around brackets. Personal preference is no brackets if not required, but do as you prefer.

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp (hunk 1 above)
Comment (angela28chen) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp]: Thanks, I didn't know this! I took it out

**Comment by angela28chen** [comment]
ready for merging

File: benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp [hunk 2]
```diff
@@ -1696,6 +1696,11 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
             frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[1], 3, &colorValues);
             break;
         }
+        case FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_TEXTURE:
+            break;
+        default: {
+            PPX_ASSERT_MSG(true, "unsupported FullscreenQuadsType: " << static_cast<int>(pFullscreenQuadsType->GetValue()));
```
Comment (apazylbe) [inline benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:1702]: Could you add a `break` in `default`? and I don't think we need `{}` there either.
The break wouldn't matter now, but if we ever added a case in the end or move the default to top we might forget to add break then

Comment (angela28chen) [reply benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp:1702]: OK, done!

**Review by hysw — APPROVED** [review]

**Review by apazylbe — APPROVED** [review]

**Pull request merged**